src/Footer/FooterContainer3/Grid1Container3.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+
+class Grid1Container3 extends Component {
```

Comment:
same here

Reply:
Yes sir, I this those things happened automatically in Vs code, I'll correct them